convert2rhel/unit_tests/rhnclassic_test.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright(C) 2016 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+import os
+
+from convert2rhel import unit_tests
+from convert2rhel import utils
+from convert2rhel import rhelvariant
+
+try:
+    import unittest2 as unittest  # Python 2.6 support
+except ImportError:
+    import unittest
+
+
+class TestUtils(unittest.TestCase):
+    class os_is_file_mock(unit_tests.MockFunction):
+        def __init__(self, return_value):
+            self.path = ""
+            self.return_value = return_value
+
+        def __call__(self, path):
+            self.path = path
+            return self.return_value
+
+    @unit_tests.mock(os.path, "isfile", os_is_file_mock(False))
+    def test_returns_successfully_if_rhn_classic_does_not_exist(self):
+        self.assertEqual(rhelvariant.check_rhn_classic(), False)
+
+    @unit_tests.mock(os.path, "isfile", os_is_file_mock(True))
+    def test_throws_if_rhn_classic_exists(self):
+        self.assertEqual(rhelvariant.check_rhn_classic(), True)
```

Comment:
Mock LGTM, but lets make `check_rhn_classic()` private, drop these tests, and write tests for `determine_rhel_variant()`. Then we can make sure that we get an exception raised by asserting

`self.assertRaises(SystemExit, rhelvariant.determine_rhel_variant)`